connector/CONFIG.md

```diff
@@ -133,20 +140,20 @@ Generates:
 - "apple tv"
 - "samsung tv"
 
-### External
+### External Sources
 
 - `external`: Array of indices you want to use.
-- [Optional] `external_source`: Accepts either `"source"` or `"target"` (See [Credentials](#credentials))
+- [Optional] `external_source`: Accepts either `"source"` or `"target"` (See [Credentials](#credentials)). ???If not specified, it uses the source of the main index.
 
 You can use an external index to populate your index.  
 This can be useful when you have an external source for analytics, for instance Google Analytics.
 
-### Expansion
+### Expansion???
```

Comment:
Query expansion (in the code, I call it `unprefixer`).
It transforms a query like `brea` into `bread`.